README.md

```diff
@@ -36,7 +36,6 @@ Git user attention
 
 3. After running `download-deps.py`.
 
-         $ cd cocos2d-x
```

Comment:
It's more like a instruction to let user know where the next command should be executed